packages/app/src/settings/SettingsContainer.cy.tsx

```diff
@@ -2,17 +2,19 @@ import { SettingsContainerFragmentDoc } from '../generated/graphql-test'
 import { defaultMessages } from '@cy/i18n'
 import SettingsContainer from './SettingsContainer.vue'
 
+const mountSettingsContainer = () => cy.mountFragment(SettingsContainerFragmentDoc, { render: (gql) => <SettingsContainer gql={gql} /> })
+
+beforeEach(() => mountSettingsContainer())
+
 describe('<SettingsContainer />', { viewportHeight: 800, viewportWidth: 900 }, () => {
-  const mountSettingsContainer = () => cy.mountFragment(SettingsContainerFragmentDoc, { render: (gql) => <SettingsContainer gql={gql} /> })
+  it('renders sections collapsed by default', () => {
+    cy.get('[data-cy="settings"]').should('be.visible')
+    cy.get('[data-cy="setting-expanded-container"]').should('not.exist')
```

Comment:
Totally up to you, personal preference
```suggestion
    cy.findByTestId('settings').should('be.visible')
    cy.findByTestId('setting-expanded-container').should('not.exist')
```